src/utils/getMembersInServer.ts

```diff
@@ -0,0 +1,20 @@
+import * as response from "../constants/responses";
+import { env } from "../typeDefinitions/default.types";
+import { DISCORD_BASE_URL } from "../constants/urls";
+
+export const getMembersInServer = async (env: env) => {
+  try {
+    const MEMBERS_URL = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}/members?limit=1000`;
```

Comment:
Is the value of the query param `limit` fixed? Or is there a possibility that we may choose to fine-tune it later on?

Reply:
`(limit is 0-1000, default:1000 )` takes another parameter `after: the highest user id in the previous page`

Reply:
the highest user id in the previous page

Reply:
https://discord.com/developers/docs/resources/guild#list-guild-members

Reply:
Create an issue to ensure for future

Reply:
Done Tracking it here #58